includes/class-wc-calypso-bridge-addons.php

```diff
@@ -72,11 +74,8 @@ public function init() {
 		}
 	}
 
-	/**
-	 * Load the class for the modified Addons menu.
-	 */
-	public function load_modified_addons_menu() {
-		require 'class-wc-modified-admin-addons.php';
+	public function get_menu_slug() {
+		return class_exists( '\Automattic\WooCommerce\Utilities\FeaturesUtil' ) && \Automattic\WooCommerce\Utilities\FeaturesUtil::feature_is_enabled( 'marketplace' ) && ! wc_calypso_bridge_is_ecommerce_trial_plan() ? 'wc-admin&path=/extensions' : 'wc-addons';
```

Comment:
We'll need to remember to change this when we eventually make the new style marketplace permanent.

Reply:
Noted! I think this would be quite relevant to how we decide to proceed with this feature checkbox. As far as I understand, we might want to:
- Permanently fix the marketplace in place and lose the checkbox, or;
- Handle the fallback which currently shows the legacy one, and instead hide it completely when the feature is disabled.

In terms of dotCom, we could force-enable the feature in all Woo Express sites (or even on all plans.)